framer/Animation.coffee

```diff
@@ -64,6 +96,9 @@ class exports.Animation extends BaseClass
 		@_originalState = @_currentState()
 		@_repeatCounter = @options.repeat
 
+	@define "layer",
+		get: -> @_layer
```

Comment:
I use later setting of layers in the animation preview of autocode. Are you ok with just using private api there?